src/pages/manageToken.tsx

```diff
@@ -87,6 +91,52 @@ export default function manageToken() {
                 </div>
             </div>
             </div>
+
+            <div className="modal fade" id="BurnTokens" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
All these modals can be converted to a single generic component and then utilise that component to render different content and actions.